internal/local/dependencies.go

```diff
@@ -64,11 +62,6 @@ func (d Dependencies) PrepareUpload() (string, error) {
 		return "", err
 	}
 
-	transpiler, err := newDefaultTranspiler()
-	if err != nil {
-		return "", err
-	}
-
 	dir, err := ioutil.TempDir("", "") // uses os.TempDir and guarantees existence and proper permissions
```

Comment:
not sure if other parts of the `PrepareUpload(...)` are important for the server-side transpilation. Should other bits be removed as well?

Reply:
I believe you can get rid of this and the `PrepareDependencies` functions completely. We no longer need to modify the archive that's provided by the user. All of this is done to transpile the js files within the provided archive, but now the new endpoint will handle that for us.